src/components/banners/EInvoiceCredits.tsx

```diff
@@ -0,0 +1,62 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2024. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+import { useCurrentCompany } from '$app/common/hooks/useCurrentCompany';
+import { useTranslation } from 'react-i18next';
+import { Banner } from '../Banner';
+import { buttonStyles } from './VerifyEmail';
+import { useQuota } from '$app/pages/settings/e-invoice/peppol/Preferences';
+import { Link } from 'react-router-dom';
+
+export const EINVOICE_CREDITS_MIN_THRESHOLD = 15;
```

Comment:
@turbo124 please adjust the min credits threshold, or let me know if we need to pull it from the API.